setup.py

```diff
@@ -67,7 +67,13 @@
 
 install_requires = [
     'Flask>=0.11.1',
-    'requests>=2.4.0',
+    # elasticsearch has hard version range dependency on requests and urllib3
+    # requests has hard version range dependency on idna
+    # Every time idna and urllib3 are updated, installation breaks because
+    # elasticsearch and requests dependencies are not resolved properly.
+    'requests>=2.4.0,<3.0.0', # from elasticsearch
+    'urllib3<1.23,>=1.21.1',  # from elasticsearch
```

Comment:
Something weird here in `elasticsearch-py`:

- v2.x has [`'urllib3>=1.8, <2.0'`](https://github.com/elastic/elasticsearch-py/blob/3d0bb532f64e1c032e5f76c68026f5357c9ce730/setup.py#L15)
- v5.x and v6.x has [`'urllib3<1.23,>=1.21.1'`](https://github.com/elastic/elasticsearch-py/blob/4189be4a2df9ef72a48e9cc59c455b6973622d9d/setup.py#L15)